src/objective-c/GRPCClient/private/GRPCHost.m

```diff
@@ -126,6 +126,13 @@ - (nullable grpc_call *)unmanagedCallWithPath:(NSString *)path
                         completionQueue:queue];
 }
 
+- (NSData *)dataWithNsString:(NSString *)string {
```

Comment:
Could you change the name to something like `nullTerminatedDataWithString:`? So that people would not think why we do not use `dataUsingEncoding`.

Reply:
Done.